docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,169 @@
+- celebrate PRs for your project.
+
+### Creating a New Highlight
+
+There are two ways to create a new highlight:
+
+1. After clicking the [Highlights tab](https://app.opensauced.pizza/feed) in the top navigation, click the input that says "Post a highlight to show your work!"
+
+2. From your profile, click the input that says "Post a highlight to show your work!"
+
+After you've clicked the input, you'll see a form that allows you to create a new highlight.
+
+There are two ways to create a highlight:
+
+1. Paste the URL to your PR, Issue, or Dev.to blog post (shown in blue below).
+2. Choose from a list of suggestions (shown in orange below).
+
+![highlight suggestions](../../static/img/highlights.png)
+
+If you're adding a blog post, you'll need to associate it to a repository by clicking the "Add a repo" button.
+
+Finally, it's time to add the details of your highlight. You can add a description yourself or use the auto-summarize function to generate a description.
+
+![gif of creating a highlight](../../static/gif/highlight.gif)
+
+### Writing an Effective Highlight
+
+Writing an effective highlight is key to showcasing your contributions and skills, and attracting potential collaborators and recruiters. Here are some tips to help you write an effective highlight:
+
+#### Highlighting Issues
+
+1. **Focus on the problem solved**: Begin by briefly describing the problem or challenge the issue addresses. This sets the context for why it was important.
+2. **Acknowledge contributor(s)**: Mention the contributor(s) who reported or worked on the issue, recognizing their effort.
+3. **Impact statement**: Explain how resolving this issue benefits the project, such as improving functionality, user experience, or security.
+
+#### Highlighting PRs
+
+1. **Summarize the changes**: Start with a clear summary of what the PR achieves. This could be a new feature, bug fix, or enhancement.
+2. **Credit the contributors**: Acknowledge everyone who contributed to the PR, including reviewers.
+3. **Highlight the value**: Explain the value of these changes to the project or its users. For instance, how it improves performance, adds a requested feature, or simplifies the codebase.
+
+#### Highlighting Dev.to Blog Posts
+
+1. **Capture the essence**: Begin with an interesting blog post summary, capturing the main theme or message.
+2. **Discuss the relevance**: Explain why this blog post is important for your open source project or community. Is it about a major update, a community event, or sharing best practices?
+3. **Engage with a call-to-action**: Encourage readers to engage with the post, whether by reading, commenting, or sharing their thoughts on the topic.
+
+## Insights: Connecting Your Repositories
+
+OpenSauced Insights you to see the impact of your contributions. You can use insights pages to track repositories' growth, analyze work, and connect with others.
+
+## Why Create an Insights Page
+
+- Personal Contribution Tracking
```

Comment:
Let's make sure this list all starts with the same part of speech. So this would be:

- Tracking personal contributions
- Building Portfolios and Developing Skills